doc/development.md

```diff
@@ -46,15 +46,14 @@ Before we can start you need to match some prerequirements on your host machine.
      #!/usr/bin/env bash
 
      export pre_onadmin_installcrowbar=$(base64 --wrap=0 <<'EOS'
-       zypper ar -f http://download.suse.de/ibs/Devel:/Cloud:/Shared:/Rubygem/SLE_11_SP3/Devel:Cloud:Shared:Rubygem.repo;
-       zypper mr -r -e sp3sdk;
+       zypper ar -f http://dist.suse.de/install/SLP/SLE-12-SDK-GM/x86_64/DVD1/ sle12-sdk
```

Comment:
It's anyway not the sense of this repo. We used the old repo for installing the ruby2.1-devel package which is now in the real SDK repo.